content/pt-br/docs/reference/glossary/service.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Service
+id: service
+date: 2021-09-17
+full_link: /docs/concepts/services-networking/service/
+short_description: >
+  Uma forma abstrata de expor uma aplicação que está executando em um conjunto de Pods como um serviço de rede.
+
+aka:
+tags:
+- fundamental
+- core-object
+---
+Uma forma abstrata de expor uma aplicação que está executando em um conjunto de {{< glossary_tooltip text="Pods" term_id="pod" >}} como um serviço de rede.
+
+<!--more-->
+
+ O conjunto de Pods referenciado por um Service é (geralmente) determinado por um {{< glossary_tooltip text="selector" term_id="selector" >}}. Se mais Pods são
```

Comment:
I would add in the glossary_tooltip text here "seletor" (translate the selector term)

Reply:
Done!